### PR TITLE
chore(google-maps):  REAME Improvement WIP

### PR DESCRIPTION
**Pull request by Ombuweb**

### CODE REVIEW

**Comment by vallemar** [comment]
I like to see that the vue examples are made with vue3 !!!!!

File: packages/google-maps/README.md [hunk 1]
```diff
@@ -1,69 +1,62 @@
-# @nativescript/google-maps
+# @nativescript/googlemap-classs
```
Comment (herefishyfish) [inline packages/google-maps/README.md]: Was this done on purpose?

File: packages/google-maps/README.md [hunk 2]
```diff
@@ -78,11 +71,9 @@ in `App_Resources/Android/AndroidManifest.xml`
...
 ```
 
-
-### iOS
-
-Modify the `Info.plist` to include the new meta tag along with your API key, the manifest is located
-in `App_Resources/iOS/Info.plist`
+**iOS**
+dd the `TNSGoogleMapsAPIKey` key and the API key as the value to the `Info.plist` file, located
+at `App_Resources/iOS`.
```
Comment (herefishyfish) [inline packages/google-maps/README.md]:
```suggestion
**iOS**

Add the `TNSGoogleMapsAPIKey` key and the API key as the value to the `Info.plist` file, located
at `App_Resources/iOS`.
```

File: packages/google-maps/README.md [hunk 3]
```diff
@@ -338,239 +436,149 @@ The Google Maps API offers five types of maps:
...
+| `geodesic` | `boolean` |
 | `strokeJointType` | JointType |
-| `visible` | boolean |
+| `visible` | `boolean` |
 | `userData` | `{ [key: string]: any }` |
 
 ## Removing Polygons
 
-You can remove a Polygon using the [GoogleMap](#google-map)s `removePolygon` function, like so: 
+You can remove a Polygon using the [GoogleMap](#googlemap-class)`s `removePolygon` function, like so:
```
Comment (herefishyfish) [inline packages/google-maps/README.md]:
```suggestion
You can remove a Polygon using the [GoogleMap](#googlemap-class)s `removePolygon` function, like so: 
```

File: packages/google-maps/README.md [hunk 4]
```diff
@@ -161,27 +160,143 @@ Vue.use(GoogleMaps)
 />
 ```
 
-## MapView API
+### Controlling the camera
+To programatically update the camera position, call the `animateCamera()` from the `GoogleMap` object.
```
Comment (herefishyfish) [inline packages/google-maps/README.md]: This should link to the CameraUpdate class in some way

**Review by herefishyfish — COMMENTED** [review]
The map class should probably be front and center still. Without the map class you're unable to use any of the functionality. It doesn't make a heap of sense to list off functionality that can't be accessed til later in the doc.

File: packages/google-maps/README.md (hunk 1 above)
Comment (Ombuweb) [reply packages/google-maps/README.md]: No. It's a mistake.

Comment (Ombuweb) [reply packages/google-maps/README.md]: Thank you very for pointing that out.

File: packages/google-maps/README.md [hunk 5]
```diff
@@ -95,32 +92,39 @@ in `App_Resources/iOS/Info.plist`
 </plist>
 ```
 
+## Installation
+```cli
+npm install @nativescript/google-mapss
+```
+
 ## Usage
+To show the map view, add the `<MapView>` element to your HTML. Then to manage the mapping features, get the reference to the [GoogleMap](#googlemap-object) instance from the `ready` event data.
```
Comment (Ombuweb) [inline packages/google-maps/README.md:101]: @herefishyfish I think by mentioning `GoogleMap` here, we can leave its API reference where it is, to maintain the documentation structure of `Installation` -> `Usage` -> `API`. What do you think?

Comment (herefishyfish) [reply packages/google-maps/README.md:101]: Yeah I'd be happy with that if we can make it more pronounced, as it stands I can see users skipping over it and it's super important. Might also be better off after the flavours though as most people will likely skip to their flavour of choice.

File: packages/google-maps/README.md [hunk 6]
```diff
@@ -95,32 +92,39 @@ in `App_Resources/iOS/Info.plist`
 </plist>
 ```
 
+## Installation
+```cli
+npm install @nativescript/google-mapss
```
Comment (herefishyfish) [inline packages/google-maps/README.md]:
```suggestion
npm install @nativescript/google-maps
```

File: packages/google-maps/README.md [hunk 7]
```diff
@@ -95,32 +92,40 @@ in `App_Resources/iOS/Info.plist`
...
+		ready="{{onReady}}"
+		mapTap="{{onTap}}"
+		mapLongPress="{{onLongPress}}"
+		markerTap="{{onMarkerTap}}"
+	/>
+</Page>
 ```
+To use the plugin in the different NativeScript flavors, modify the `main.ts` to import and then register it.
 
+> **Note** To handle the map features, see the [GoogleMap object](#googlemap-object) API.
```
Comment (Ombuweb) [inline packages/google-maps/README.md:123]: @herefishyfish  I think adding that note at the end of each flavor's example would be helpful.

**Pull request merged**